lib/runner.js

```diff
@@ -726,8 +747,9 @@ Runner.prototype.runSuite = function(suite, fn) {
   }
 
   this.emit(constants.EVENT_SUITE_BEGIN, (this.suite = suite));
+  suite.started = true;
```

Comment:
if we have any more booleans on `Suite` like this, we should consider a `state` prop instead